elastic/kibana
# [Uptime] Use data streams for configs being passed to service

## Summary

Fixes https://github.com/elastic/kibana/issues/121402

Use data stream format for heartbeat configs.

### Testing

- Make sure monitor management is working as expected
- Make sure data is being stored in `synthetics-*` streams
- Remove heartbeat-* from uptime indices settings and just leaves `synthetics-* `and make sure all the added monitors are being displayed
